contrib/storage_plugin/README_zh_CN.MD

```diff
@@ -3,27 +3,28 @@
 A tool to manage external storage in PAI.
```

Comment:
Use `.md` in filename instead of `.MD`?

Reply:
the name is generated by tool, and exactly same as English file name. to get consistence, the English file name should be changed firstly.